R/app.R

```diff
@@ -72,7 +76,8 @@
 #' }
 #' @export
 shinyApp <- function(ui, server, onStart=NULL, options=list(),
-                     uiPattern="/", enableBookmarking=NULL) {
+                     uiPattern="/", enableBookmarking=NULL,
+                     bs_theme=getShinyOption("bs_theme")) {
```

Comment:
I was originally thinking `bs_theme` to avoid name clashing with `bootstrapPage()`'s `theme`, but I'm coming around to `theme`, especially if `theme` in rmarkdown winds up accepting both a theme name or a full-blown theme object. Also, I think bootstraplib's args will be named `theme`

Reply:
TODO: should we default this option to `bs_global_theme()`?